cmd/flags.go

```diff
@@ -50,7 +42,7 @@ func StartPprof() {
 		go func() {
 			log.Debugf("Starting pprof page at http://%s/debug/pprof", *PprofAddr)
 			if err := http.ListenAndServe(*PprofAddr, nil); err != nil {
-				log.Errorf("Unable to start PPROF HTTP interface: %v", err)
+				_ = log.Errorf("Unable to start PPROF HTTP interface: %v", err)
```

Comment:
Is this to pacify some linting tool?  It's pretty typical to ignore errors for logging calls and this results in inconsistent style.  It's not a big issue though.

Reply:
@hwh33 yep it was for linting. I personally don't like functions that ignore returned errors, explicit is better than implicit and all that jazz ;)